Source/Scene/Scene.js

```diff
@@ -3024,7 +3025,9 @@ define([
 
     var scratchPickPositionCartographic = new Cartographic();
 
-    /**
+    var pickPositionCache={};
```

Comment:
In the case of multiple scenes, this may be better as a private variable of `Scene`.